### DISCUSSION TITLE
Welcome to Linux Mint Discussions!

### DISCUSSION THREAD

**Discussion by clefebvre** [linuxmint/discussions#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by ghost** [comment]
Hey there!

**Reply by LinuxOnTheDesktop** [reply]
Well hello!

**Reply by Geekman75** [reply]
Hello, i was wondering why there is no way on linux mint to economize energy batery like ubuntu does and soon zorin os 17 ?

**Reply by BossFloyd** [reply]
Good evening, you can try installing auto-cpufreq. I'm runninng LMDE. It's working from what I can see been running on my laptop for about 2 weeks. It helps.

**Comment by LinuxOnTheDesktop** [comment]
Is the idea that we discuss things here on this very page? I _think_ it is, but I'd like to check!

**Comment by Geekman75** [comment]
Hello, i was wondering why there is no way on linux mint to economize energy batery like ubuntu does and soon zorin os 17 ?

**Reply by anaximeno** [reply]
@Geekman75 are you on Linux Mint or LMDE?

**Reply by Geekman75** [reply]
Hello, I'm on Linux Mint.

**Comment by m9789959957** [comment]
Why Linux mint 21.3 version is having live session user as linux and $HOME as /home/linux though it is mentioned as mint in Linux Mint official web page ?

**Comment by scriptura17** [comment]
Hello everyone ! Today I install Linux mint 21.3 on my old imac 20inch 2008. 
The wifi doesn't work can anyone help me please ?

**Reply by anaximeno** [reply]
Try to connect your device to a wired connection (from ethernet, or share through your phone) and open the driver manager application to install the Broadcom drivers for your wifi then reboot your system.

**Comment by billykd4dpb** [comment]
hello everyone i noticed a possible bug with eclipside and mint there seems to be a problem in maybe nemo not being able to generate windows or something

**Reply by billykd4dpb** [reply]
when i ran eclipse on ubuntu  it worked flowless

**Comment by 01Mehdi10** [comment]
Hello,
I have upgraded some days ago to 21.3. Since then, android mobile and windows system can't connect to the hotspot when set wpa/wpa2.
Here are the system settings : 
System:
  Kernel: 5.15.0-106-generic x86_64 bits: 64 compiler: gcc v: 11.4.0 Desktop: Cinnamon 6.0.4
    tk: GTK 3.24.33 wm: muffin vt: 7 dm: LightDM 1.30.0 Distro: Linux Mint 21.3 Virginia
    base: Ubuntu 22.04 jammy
Machine:
  Type: Desktop System: LENOVO product: 10120 v: IdeaCentre K450 serial: <superuser required>
  Mobo: LENOVO model: N/A v: 31900058 STD serial: <superuser required> UEFI: LENOVO v: I1KT34AUS
    date: 11/18/2013
Battery:
  Device-1: hidpp_battery_0 model: Logitech Wireless Keyboard K230 serial: <filter>
    charge: 100% (should be ignored) rechargeable: yes status: Discharging
  Device-2: hidpp_battery_1 model: Logitech M215 serial: <filter>
    charge: 55% (should be ignored) rechargeable: yes status: Discharging
CPU:
  Info: quad core model: Intel Core i5-4440 bits: 64 type: MCP smt: <unsupported> arch: Haswell
    rev: 3 cache: L1: 256 KiB L2: 1024 KiB L3: 6 MiB
  Speed (MHz): avg: 2563 high: 3049 min/max: 800/3300 cores: 1: 2006 2: 3049 3: 2297 4: 2902
    bogomips: 24743
  Flags: avx avx2 ht lm nx pae sse sse2 sse3 sse4_1 sse4_2 ssse3 vmx
Graphics:
  Device-1: NVIDIA GK107 [GeForce GTX 650] vendor: Micro-Star MSI driver: nvidia v: 470.239.06
    pcie: speed: 8 GT/s lanes: 16 bus-ID: 01:00.0 chip-ID: 10de:0fc6 class-ID: 0300
  Display: x11 server: X.Org v: 1.21.1.4 driver: X: loaded: nvidia
    unloaded: fbdev,modesetting,nouveau,vesa gpu: nvidia display-ID: :0 screens: 1
  Screen-1: 0 s-res: 1920x1080 s-dpi: 95 s-size: 513x292mm (20.2x11.5") s-diag: 590mm (23.2")
  Monitor-1: DVI-I-0 res: 1920x1080 hz: 60 dpi: 102 size: 477x268mm (18.8x10.6")
    diag: 547mm (21.5")
  OpenGL: renderer: NVIDIA GeForce GTX 650/PCIe/SSE2 v: 4.6.0 NVIDIA 470.239.06
    direct render: Yes
Audio:
  Device-1: Intel 8 Series/C220 Series High Definition Audio vendor: Lenovo driver: snd_hda_intel
    v: kernel bus-ID: 00:1b.0 chip-ID: 8086:8c20 class-ID: 0403
  Device-2: NVIDIA GK107 HDMI Audio vendor: Micro-Star MSI driver: snd_hda_intel v: kernel pcie:
    speed: 8 GT/s lanes: 16 bus-ID: 01:00.1 chip-ID: 10de:0e1b class-ID: 0403
  Sound Server-1: ALSA v: k5.15.0-106-generic running: yes
  Sound Server-2: PulseAudio v: 15.99.1 running: yes
  Sound Server-3: PipeWire v: 0.3.48 running: yes
Network:
  Device-1: Realtek RTL8111/8168/8411 PCI Express Gigabit Ethernet vendor: Lenovo driver: r8169
    v: kernel pcie: speed: 2.5 GT/s lanes: 1 port: d000 bus-ID: 03:00.0 chip-ID: 10ec:8168
    class-ID: 0200
  IF: enp3s0 state: up speed: 100 Mbps duplex: full mac: <filter>
  Device-2: Realtek RTL8188EE Wireless Network Adapter vendor: Lenovo driver: rtl8188ee
    v: kernel pcie: speed: 2.5 GT/s lanes: 1 port: c000 bus-ID: 04:00.0 chip-ID: 10ec:8179
    class-ID: 0280
  IF: wlp4s0 state: up mac: <filter>
Drives:
  Local Storage: total: 931.51 GiB used: 203.84 GiB (21.9%)
  ID-1: /dev/sda vendor: Western Digital model: WD10EZEX-08M2NA0 size: 931.51 GiB
    speed: 6.0 Gb/s type: HDD rpm: 7200 serial: <filter> rev: 1A01 scheme: GPT
Partition:
  ID-1: / size: 451.36 GiB used: 203.81 GiB (45.2%) fs: ext4 dev: /dev/sda7
  ID-2: /boot/efi size: 96 MiB used: 31.5 MiB (32.8%) fs: vfat dev: /dev/sda2
Swap:
  ID-1: swap-1 type: file size: 2 GiB used: 524 KiB (0.0%) priority: -2 file: /swapfile
USB:
  Hub-1: 1-0:1 info: Full speed or root hub ports: 2 rev: 2.0 speed: 480 Mb/s chip-ID: 1d6b:0002
    class-ID: 0900
  Hub-2: 1-1:2 info: Intel Integrated Rate Matching Hub ports: 6 rev: 2.0 speed: 480 Mb/s
    chip-ID: 8087:8008 class-ID: 0900
  Hub-3: 2-0:1 info: Full speed or root hub ports: 2 rev: 2.0 speed: 480 Mb/s chip-ID: 1d6b:0002
    class-ID: 0900
  Hub-4: 2-1:2 info: Intel Integrated Rate Matching Hub ports: 6 rev: 2.0 speed: 480 Mb/s
    chip-ID: 8087:8000 class-ID: 0900
  Hub-5: 3-0:1 info: Hi-speed hub with single TT ports: 12 rev: 2.0 speed: 480 Mb/s
    chip-ID: 1d6b:0002 class-ID: 0900
  Device-1: 3-1:2 info: Logitech Unifying Receiver type: Keyboard,Mouse,HID
    driver: logitech-djreceiver,usbhid interfaces: 3 rev: 2.0 speed: 12 Mb/s power: 98mA
    chip-ID: 046d:c52b class-ID: 0300
  Device-2: 3-2:3 info: Logitech Unifying Receiver type: Mouse,HID
    driver: logitech-djreceiver,usbhid interfaces: 2 rev: 2.0 speed: 12 Mb/s power: 98mA
    chip-ID: 046d:c52f class-ID: 0300
  Device-3: 3-7:4 info: Samsung Galaxy A5 (MTP) type: Still Imaging,Abstract (modem),CDC-Data
    driver: cdc_acm,usbfs interfaces: 4 rev: 2.1 speed: 480 Mb/s power: 500mA chip-ID: 04e8:6860
    class-ID: 0a00 serial: <filter>
  Device-4: 3-10:5 info: Realtek RTS5182 Card Reader type: Mass Storage driver: ums-realtek
    interfaces: 1 rev: 2.0 speed: 480 Mb/s power: 500mA chip-ID: 0bda:0184 class-ID: 0806
    serial: <filter>
  Hub-6: 4-0:1 info: Super-speed hub ports: 6 rev: 3.0 speed: 5 Gb/s chip-ID: 1d6b:0003
    class-ID: 0900
Sensors:
  System Temperatures: cpu: 29.8 C mobo: 27.8 C gpu: nvidia temp: 35 C
  Fan Speeds (RPM): N/A gpu: nvidia fan: 14%
Repos:
  Packages: 3215 apt: 3204 flatpak: 11
  No active apt repos in: /etc/apt/sources.list
  Active apt repos in: /etc/apt/sources.list.d/official-package-repositories.list
    1: deb http: //packages.linuxmint.com virginia main upstream import backport
    2: deb http: //archive.ubuntu.com/ubuntu jammy main restricted universe multiverse
    3: deb http: //archive.ubuntu.com/ubuntu jammy-updates main restricted universe multiverse
    4: deb http: //archive.ubuntu.com/ubuntu jammy-backports main restricted universe multiverse
    5: deb http: //security.ubuntu.com/ubuntu/ jammy-security main restricted universe multiverse
  Active apt repos in: /etc/apt/sources.list.d/teamviewer.list
    1: deb [signed-by=/usr/share/keyrings/teamviewer-keyring.gpg] https: //linux.teamviewer.com/deb stable main
  Active apt repos in: /etc/apt/sources.list.d/vscode.list
    1: deb [arch=amd64,arm64,armhf] http: //packages.microsoft.com/repos/code stable main
  Active apt repos in: /etc/apt/sources.list.d/winehq-bullseye.sources
    1: deb [arch=amd64 i386] https: //dl.winehq.org/wine-builds/debian bullseye main
  Active apt repos in: /etc/apt/sources.list.d/winehq-focal.sources
    1: deb [arch=amd64 i386] https: //dl.winehq.org/wine-builds/ubuntu focal main
Info:
  Processes: 241 Uptime: 37m wakeups: 6 Memory: 3.75 GiB used: 1.95 GiB (51.9%) Init: systemd
  v: 249 runlevel: 5 Compilers: gcc: 11.4.0 alt: 11/12/9 Client: Cinnamon v: 6.0.4 inxi: 3.3.13

Thanks  for your help

**Reply by anaximeno** [reply]
This might not be the best place to find help for this, go to the https://linuxmint.com/ website and click on the `Links` section on the navigation bar, there you'll see some places where you might find support like the Forums, and Discord or Reddit groups (and others). It might be easier to find support there.

**Reply by 01Mehdi10** [reply]
Ok thanks for the information. Just to add one more thing in case it's useful here. The hotspot works without security.

**Comment by mrpauloen** [comment]
Hi everyone.  
I'm not completely new to Linux, but I'm just starting out with GitHub.  
I wonder why we can't continue to submit new features in the Nemo repository: [https://github.com/linuxmint/nemo/blob/master/.github/ISSUE_TEMPLATE/feature_request.md](url)

**Reply by 100savage** [reply]
From this page
https://github.com/linuxmint/nemo/tree/master/.github/ISSUE_TEMPLATE

Feature requests are no longer accepted here, please use 
https://github.com/orgs/linuxmint/discussions instead.

**Comment by Reno-Sifana** [comment]
Hi. Since about 1 week ago, Firefox 135 already been released on February 4, 2025, [See Firefox 135 release notes.](https://www.mozilla.org/en-US/firefox/135.0/releasenotes/) 
![Screenshot from 2025-02-15 08-41-17](https://github.com/user-attachments/assets/bae8d000-7bc2-466c-a98e-db54db5d490a)
 But unfortunately, Firefox on Linux Mint's Repositories still on version 134.0.2 until now (February 15, 2025). 
![Screenshot from 2025-02-15 08-40-40](https://github.com/user-attachments/assets/d09bbca0-d293-4990-a36d-5e8814499832)
 I hope @clefebvre will build Firefox 135.0 soon and uploaded to Linux Mint's Repositories. And I can get updates of Firefox from local mirror. (of course, I'm always check for updates periodically).

**Comment by webaborigen** [comment]
Hi! I recently encountered an issue updating Flatpaks in the Cinnamon application manager. It’s also impossible to install Flatpaks via the terminal. Every installation attempt is interrupted with a 403 error. The packages are being requested from the Amazon cloud. I specify an alternative source in the application manager, and the APT cache updates, but it still tries to download Flatpak from Amazon and I get the same 403 error. Any advice would be appreciated. Reinstalling the system doesn't help

**Reply by Sadia2000** [reply]
Pretty sure Flatpak does not rely on APT cache. I cannot say anything else other than that, so yeah...

**Comment by gandb** [comment]
Hey there, I want put a review for all applications I installed in my Mint but the https://community.linuxmint.com/auth/register ask me a code, what is this code and how I can to get this?

**Reply by timfeierabend** [reply]
You need to register with the [Linux Mint forums](https://forums.linuxmint.com/) first.

From the community registration page:
> To prevent SPAM bots from harming our community, we now require a registration code. Login to the Linux Mint forums and go to your user control panel. Your registration code will be written at the bottom of the panel.

**Comment by JosephMcc** [comment]
Whoops! Sorry for closing this. Was deleting some spam.
